### PR TITLE
Feature/cms page priorities

### PR DESCRIPTION
**Pull request by lorenas**
Adds priority parameter for CMS page entity.

### CODE REVIEW

File: Entity/CmsPage.php [hunk 1]
```diff
@@ -163,6 +163,12 @@ class CmsPage
     private $locale;
 
     /**
+     * @var integer
+     * @ORM\Column(name="priority", type="integer", nullable=true)
```
Comment (sanis) [inline Entity/CmsPage.php]: nullable is not needed to be = true when you are setting default value.

**Review by sanis — APPROVED** [review]

**Comment by N3m1s** [comment]
So, what is the point of adding just one property to the entity? Is there any additional functionality regarding this? Why it's not presented?

File: Entity/CmsPage.php [hunk 2]
```diff
@@ -497,4 +503,20 @@ public function setImageAlt($imageAlt)
...
+    {
+        return $this->priority;
+    }
+
+    /**
+     * @param int $priority
+     */
+    public function setPriority($priority)
+    {
+        $this->priority = $priority;
```
Comment (N3m1s) [inline Entity/CmsPage.php]: We tend to use fluent setters.

**Review by N3m1s — APPROVED** [review]

**Comment by lorenas** [comment]
@N3m1s @linasmo @sanis Would you have any further comments on this feature?

**Review by N3m1s — APPROVED** [review]

**Pull request merged**